docs/operating-scylla/procedures/cluster-management/replace-dead-node.rst

```diff
@@ -68,11 +68,11 @@ Procedure
 
     - **rpc_address** - Address for client connection (Thrift, CQL)
 
-#. Add the ``replace_address_first_boot`` parameter to the ``scylla.yaml`` config file on the new node. This line can be added to any place in the config file. After a successful node replacement, there is no need to remove it from the ``scylla.yaml`` file. (Note: The obsolete parameter "replace_address" is not supported and should not be used). The value of the ``replace_address_first_boot`` parameter should be the IP address of the node to be replaced.
+#. Add the ``replace_node_first_boot`` parameter to the ``scylla.yaml`` config file on the new node. This line can be added to any place in the config file. After a successful node replacement, there is no need to remove it from the ``scylla.yaml`` file. (Note: The obsolete parameters "replace_address" and "replace_address_first_boot" are not supported and should not be used). The value of the ``replace_node_first_boot`` parameter should be the Host ID of the node to be replaced.
 
     For example (using the address of the failed node from above):
 
-    ``replace_address_first_boot: 192.168.1.203``   
+    ``replace_node_first_boot: 87a2b646-c968-4108-a38b-f8468b8b7360``
```

Comment:
@bhalevy 
- I understand that Host ID should be used, but I'm not sure why this one is. The Host ID of the dead node in the example (address 192.168.1.203) is 675ed9f4-6564-6dbd-can8-43fddce952gy.
-I think that "address" on line 73 should be replaced with "Host ID".

Reply:
> @bhalevy
> 
> * I understand that Host ID should be used, but I'm not sure why this one is. The Host ID of the dead node in the example (address 192.168.1.203) is 675ed9f4-6564-6dbd-can8-43fddce952gy.

good point

> * I think that "address" on line 73 should be replaced with "Host ID".

Right, thanks